openmc/settings.py

```diff
@@ -1087,3 +1136,88 @@ def export_to_xml(self):
         tree = ET.ElementTree(self._settings_file)
         tree.write("settings.xml", xml_declaration=True,
                              encoding='utf-8', method="xml")
+
+
+class ResonanceScattering(object):
+    """Specification of the elastic scattering model for resonant isotopes.
+
+    Attributes
+    ----------
+    nuclide : openmc.nuclide.Nuclide
+        The nuclide affected by this resonance scattering treatment.
+    nuclide_0K : openmc.nuclide.Nuclide
+        This should be the same isotope as `nuclide`, but it should have an
```

Comment:
We haven't used the "`" operator to surround parameters in docstrings elsewhere in the API. Is there a good reason to do so? If so, we can keep them here and aim to put them in the docstrings elsewhere. Otherwise, they should be removed here (personally I find them clunky to parse in the docstring itself, but perhaps they improve the rendering in sphinx).